botocore/signers.py

```diff
@@ -56,7 +57,11 @@ def __init__(self, service_name, region_name, signing_name,
         self._signing_name = signing_name
         self._signature_version = signature_version
         self._credentials = credentials
-        self._event_emitter = event_emitter
+
+        self._event_emitter = None
+        if event_emitter is not None:
```

Comment:
Quick question: are there cases when the passed in `event_emitter` can be `None`?  I don't think this is possible is it?